pyClient/zeth/encryption.py

```diff
@@ -217,3 +234,23 @@ def decrypt(
     message = decryptor.update(ct_sym)
 
     return message
+
+
+def _exchange(sk: EncryptionSecretKey, pk: EncryptionPublicKey) -> bytes:
```

Comment:
What does `_` mean? Does it mean "internal function"?

Reply:
(Same question for the `_` prefix used for the constants in the tests)

Reply:
Yes, exactly.  The linters will complain if another module tries to use it, and I think they also use this to check for unused functions.

Reply:
I see, thanks